templates/cinder/config/cinder.conf

```diff
@@ -1,8 +1,11 @@
 [DEFAULT]
 # transport_url = rabbit://openstack:RABBIT_PASS@controller
 auth_strategy = keystone
-# TODO
-glance_api_servers=http://glanceapi.openstack.svc:9292/
+# TODO: Decide if we want the operator to generate the api servers, which is
+#       more efficient when creating volumes from image (no keystone requests).
+#       For now rely on checking the catalog info
+#       glance_api_servers=http://glanceapi.openstack.svc:9292/
+glance_catalog_info value = image:glance:internalURL
```

Comment:
Is `value` supposed to be here?  Pardon my ignorance if it is, it just seems like it's breaking the usual format

Reply:
No, it should not, and the service run fine so I only gave a quick glance at the service start where oslo.config showed that it's using the default value (`image:glance:publicURL`) instead of the one that I was trying to set.

Reply:
Fixed it in https://github.com/openstack-k8s-operators/cinder-operator/pull/36